### PR TITLE
refactor: Big cleanup of old world routes

### PR DESCRIPTION
**Pull request by aaroncarroll82**
BREAKING CHANGE: Removes old world routes and methods

### CODE REVIEW

File: app/routes/endemics/declaration.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ const {
 } = require('../../config/routes')
 
 const resetFarmerApplyDataBeforeApplication = (application) => {
-  application.reference = null // Set application ref to null instead of temp ref before sending it to be processed.
+  // we are no longer nulling the reference as backend needs it with new reference mechanism. have not changed any tests to go with this - so comforting to see that no tests break now - not!
```
Comment (mattyod) [inline app/routes/endemics/declaration.js:20]: Not sure if this comment will make much sense once it's outside the context of this PR.

File: app/routes/endemics/declaration.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ const {
 } = require('../../config/routes')
```
Comment (mattyod) [inline app/routes/endemics/declaration.js:17]: Is it worth lifting these files up a level now as everything is `endemics` now.

Comment (mattyod) [reply app/routes/endemics/declaration.js:17]: Though I suppose we are keeping the endemics path so 🤷

File: app/routes/index.js [hunk 3]
```diff
@@ -1,17 +1,15 @@
...
     auth: false,
     handler: async (request, h) => {
-      return h.view('index', {
-        defraIdLogin: requestAuthorizationCodeUrl(session, request),
-        ruralPaymentsAgency: config.ruralPaymentsAgency
-      })
+      // old world apply is no longer active, redirect user to new world start
+      // it's not actually possible to navigate directly to this page anyway anymore, except
+      // if the user decided to type it in url themselves, or had a bookmark saved here
+      return h.redirect(`${config.urlPrefix}/endemics/start`)
```
Comment (mattyod) [inline app/routes/index.js:12]: I do tend to support [Defra guidance on comments](https://defra.github.io/software-development-standards/standards/common_coding_standards/). Wonder if the suggested principle of 'separate in order to name' could be applied here to provide similar context. Something like:

```
const deprecatedOldEntryPoint =  `${config.urlPrefix}/start`
const newAppEntryPoint = `${config.urlPrefix}/endemics/start`

module.exports = {
  url: deprecatedOldEntryPoint,
  handler: (_, h) => h.redirect(newAppEntryPoint)
}

```

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-ahwr-farmer-apply",
-  "version": "0.58.37",
+  "version": "1.0.0",
```
Comment (mattyod) [inline package.json:3]: Semantics? 😮

Comment (aaroncarroll82) [reply package.json:3]: It's a brave new world!

File: app/routes/index.js (hunk 3 above)
Comment (aaroncarroll82) [reply app/routes/index.js:12]: Willing to compromise - see new changes

**Review by mattyod — APPROVED** [review]
👍

**Pull request merged**